examples/ema_cross.py

```diff
@@ -16,45 +18,45 @@ def onEnter(self, update):
     emaS = self.indicators['emaS']
     s = iv['emaS']
     l = iv['emaL']
-
     if emaS.crossed(l):
       if s > l:
-        return self.openLongPositionMarket({
-          'mtsCreate': update['mts'],
-          'price': update['price'],
-          'amount': 1,
-        })
+        try:
+          self.openLongPositionMarket(
+              mtsCreate=update['mts'], price=update['price'], amount=1)
+        except PositionError as e:
+          logging.error(e)
       else:
-        return self.openShortPositionMarket({
-          'mtsCreate': update['mts'],
-          'price': update['price'],
-          'amount': 1,
-        })
+        try:
+          self.openShortPositionMarket(
+              mtsCreate=update['mts'], price=update['price'], amount=1)
+        except PositionError as e:
+          logging.error(e)
 
   def onUpdateShort(self, update):
     iv = self.indicatorValues()
     s = iv['emaS']
     l = iv['emaL']
-
-    if s < l:
-      return self.closePositionMarket({
-        'mtsCreate': update['mts'],
-        'price': update['price']
-      })
+    if s > l:
+      try:
+        self.closePositionMarket(
+          mtsCreate=update['mts'], price=update['price'])
```

Comment:
Hmm, what do you mean by "typing"?

Reply:
Well I thought in terms of Documentation as code, it would be better if we explicity defined the parameters like this rather than passing around a json object. Personally It took me a while to figure out what kind of data gets thrown into the order params. But I can also see why we would stick with the json object since it is consistent with the nodejs version.